lib/steps/follow_the_link.rb

```diff
@@ -0,0 +1,59 @@
+module Spreewald
+  module Steps
+    class FollowTheLink
+      class NoVisitableLinkFound < StandardError
+        def initialize(paths, index)
+          error_message = <<~MESSAGE
+            Could not follow the #{index} link in the email.
+          MESSAGE
+          if paths&.empty?
+            error_message << "There are no found link paths in the email."
+          else
+            error_message << "These are the found link paths: #{paths.join(', ')}"
```

Comment:
`Found no ... in the email` sound way nicer, thx.
I'm not sure about `links` though, since it actually only displays the paths (e.g `/abc/def`) , could be a little bit confusing.